lib/rules/no-duplicate-selectors/index.js

```diff
@@ -13,7 +13,7 @@ const resolvedNestedSelector = require("postcss-resolve-nested-selector")
 const ruleName = "no-duplicate-selectors"
 
 const messages = ruleMessages(ruleName, {
-  rejected: selector => `Unexpected duplicate selector "${selector}"`,
+  rejected: (selector, firstDuplicateLine) => `Unexpected duplicate selector "${selector}" is also at line ${firstDuplicateLine}`,
```

Comment:
Let's change this wording to `Unexpected duplicate of selector "${selector}" at line ${firstDuplicateLine}`.